models/Registro_plan_adquisiciones.go

```diff
@@ -10,22 +10,24 @@ import (
-	AreaFuncional          int                `orm:"column(area_funcional)"`
-	CentroGestor           int                `orm:"column(centro_gestor)"`
-	FechaCreacion          string             `orm:"column(fecha_creacion);type(timestamp without time zone)"`
-	FechaModificacion      string             `orm:"column(fecha_modificacion);type(timestamp without time zone)"`
-	ResponsableId          int                `orm:"column(responsable_id)"`
-	Activo                 bool               `orm:"column(activo)"`
-	MetaId                 string             `orm:"column(meta_id);null"`
-	ProductoId             string             `orm:"column(producto_id);null"`
-	RubroId                string             `orm:"column(rubro_id)"`
-	FechaEstimadaInicio    string             `orm:"column(fecha_estimada_inicio);type(timestamp without time zone)"`
-	FechaEstimadaFin       string             `orm:"column(fecha_estimada_fin);type(timestamp without time zone)"`
-	FuenteFinanciamientoId string             `orm:"column(fuente_financiamiento_id)"`
-	ActividadId            int                `orm:"column(actividad_id)"`
-	ValorActividad         int                `orm:"column(valor_actividad)"`
-	PlanAdquisicionesId    *PlanAdquisiciones `orm:"column(plan_adquisiciones_id);rel(fk)"`
+	Id                         int                `orm:"column(id);pk;auto"`
+	AreaFuncional              int                `orm:"column(area_funcional)"`
+	CentroGestor               int                `orm:"column(centro_gestor)"`
+	FechaCreacion              string             `orm:"column(fecha_creacion);type(timestamp without time zone)"`
+	FechaModificacion          string             `orm:"column(fecha_modificacion);type(timestamp without time zone)"`
+	ResponsableId              int                `orm:"column(responsable_id)"`
+	Activo                     bool               `orm:"column(activo)"`
+	MetaId                     string             `orm:"column(meta_id);null"`
+	ProductoId                 string             `orm:"column(producto_id);null"`
+	PlanAdquisicionesId        *PlanAdquisiciones `orm:"column(plan_adquisiciones_id);rel(fk)"`
+	RubroId                    string             `orm:"column(rubro_id)"`
+	FechaEstimadaInicio        string             `orm:"column(fecha_estimada_inicio);type(timestamp without time zone)"`
+	FechaEstimadaFin           string             `orm:"column(fecha_estimada_fin);type(timestamp without time zone)"`
+	FuenteFinanciamientoId     string             `orm:"column(fuente_financiamiento_id)"`
+	ActividadId                int                `orm:"column(actividad_id)"`
+	ValorActividad             int                `orm:"column(valor_actividad)"`
+	FechaEstimadaOfertasInicio string             `orm:"column(fecha_estimada_ofertas_inicio);type(timestamp without time zone)"`
+	FechaEstimadaOfertasFin    string             `orm:"column(fecha_estimada_ofertas_fin);type(timestamp without time zone)"`
```

Comment:
De acuerdo a la migración (`database/scripts/20220718_210029_fecha_estimada_ofertas_up.sql`) Revisar si funciona igual - [Referencia](https://beego.vip/docs/mvc/model/models.md#postgresql)
```suggestion
	FechaEstimadaOfertasInicio string             `orm:"column(fecha_estimada_ofertas_inicio)"`
	FechaEstimadaOfertasFin    string             `orm:"column(fecha_estimada_ofertas_fin)"`
```
Si esta alternativa también funciona, mejor aún (también de acuerdo a la referencia):
```suggestion
	FechaEstimadaOfertasInicio time.Time          `orm:"column(fecha_estimada_ofertas_inicio)"`
	FechaEstimadaOfertasFin    time.Time          `orm:"column(fecha_estimada_ofertas_fin)"`
```